README.md

```diff
@@ -1,3 +1,20 @@
 # RIPE Bus API for Javascript
 
 The Javascript API for the RIPE Bus.
+
+## Initialization 
+
+```javascript
+import { API as RipeBusAPI } from "ripe-bus-api-js";
+
+await RipeBusAPI.load();
+const bus = new RipeBusAPI();
+```
+
+## Usage
+
+```javascript
+bus.bind("order", (message) => { ... })
```

Comment:
separate bind from trigger and make the second parameter either a function or an Object